cinder/volume/drivers/vmware/vmdk.py

```diff
@@ -688,7 +711,6 @@ def _select_datastore(self, req, host=None):
                 LOG.error("There are no valid hosts available in "
                           "configured cluster(s): %s.", self._clusters)
                 raise vmdk_exceptions.NoValidHostException()
-
```

Comment:
What that do?

Reply:
This was just a line difference between train and queens when I applied my datastore patch.  I'll put it back in.